stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/plugin/listener/TypingEventListenerImpl.kt

```diff
@@ -145,7 +145,7 @@ internal class TypingEventListenerImpl(
             when (eventType) {
                 EventType.TYPING_START ->
                     channelState.keystrokeParentMessageId =
-                        extraData.getOrDefault(ARG_TYPING_PARENT_ID, null)?.toString()
+                        extraData[ARG_TYPING_PARENT_ID] as? String
```

Comment:
Safe casting here because  

```kotlin
val someNewString = someNullableString.toString()
```

does not preserve the information as `null` value but rather results in a non-null string value of "null".